### DISCUSSION TITLE
iOS App Transfer - Will FCM push notification ability be lost?

### DISCUSSION THREAD

**Discussion by dooleyb1** [invertase/react-native-firebase#6190] [Q&A]
Hello everyone,

I’m considering doing an [iOS App Transfer to another App Store Connect account](https://help.apple.com/app-store-connect/#/deved688524f) however I am concerned about losing push notification ability to iOS users via FCM during the transition. I’ve had a dig around but couldn’t find any documentation directly confirming whether this would / wouldn’t be the case.

Has anyone done this before / does anyone know if we will lose the ability to send push notifications to users who may be on the old build after we have made the app transfer?

Thanks! :smile:

**Comment by mikehardy** [comment]
Based on it maintaining the same Bundle ID, and I assume then the same actual installation of the app on devices, I do not think the app would get a new "installation id" and FCM token. So the same app install should be addressable by the same token and thus messaging should be okay.

You might be well served to ask this (search first though...) in firebase-ios-sdk repo and/or on stackoverflow, and if it's really critical and means a lot of revenue, it may actually justify testing it if no one can provide an authoritative answer

**Reply by dooleyb1** [reply]
Thanks for the response @mikehardy . I'll have a search through the `firebase-ios-sdk` repo & SA and open the question there if I can't find any answers. Shall report back here with any findings also 👍
